### PR TITLE
Refactor env variables

### PR DESCRIPTION
**Pull request by mauriziovitale**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: protractor.conf.js [hunk 1]
```diff
@@ -21,13 +21,13 @@ const width = 1366;
 const height = 768;
 
 const SAVE_SCREENSHOT = process.env.SAVE_SCREENSHOT === 'true';
-const API_CONTENT_HOST = process.env.API_CONTENT_HOST || 'http://localhost:8080';
+const APP_CONFIG_ECM_HOST = process.env.APP_CONFIG_ECM_HOST || 'http://localhost:8080';
```
Comment (adinapitul) [inline protractor.conf.js:24]: Shouldn't this be just API_HOST like it is on adw and apps?

File: protractor.conf.js [hunk 2]
```diff
@@ -21,13 +21,13 @@ const width = 1366;
...
 const MAXINSTANCES = process.env.MAXINSTANCES || 1;
 const MAX_RETRIES = process.env.MAX_RETRIES || 1;
 const LOG_LEVEL = process.env.LOG_LEVEL || 'ERROR';
 
 const appConfig = {
-  hostEcm: API_CONTENT_HOST,
+  hostEcm: APP_CONFIG_ECM_HOST,
```
Comment (adinapitul) [inline protractor.conf.js:30]: Same as before, should be API_HOST

File: protractor.conf.js (hunk 1 above)
Comment (mauriziovitale) [reply protractor.conf.js:24]: We are still discussing that. For now, there are different variables for ecm and bpm.

Comment (mauriziovitale) [reply protractor.conf.js:24]: @popovicsandras am I correct ?

Comment (popovicsandras) [reply protractor.conf.js:24]: @adinapitul We are changing and standardising these variables in every repository at the moment. What you can see here is going to be the new name of variables, which is going to be accross every repo, and with this we even get rid of the variable name differences between local dev and deployement time. (We are going to use the same var names as what the docker container needs)

**Review by popovicsandras — APPROVED** [review]

**Pull request merged**